server/server.go

```diff
@@ -258,7 +258,7 @@ func (s *Server) loadSandbox(id string) error {
 		return err
 	}
 	if err = s.runtime.UpdateStatus(scontainer); err != nil {
-		return fmt.Errorf("error updating status for pod sandbox infra container %s: %v", scontainer.ID(), err)
+		logrus.Warnf("error updating status for pod sandbox infra container %s: %v", scontainer.ID(), err)
```

Comment:
it'll re-create the pod if status != READY - dockershim behaves the same, if the container/sandbox isn't running (like after a reboot), it returns status UNKNOWN and kube will restart/recreate everything https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/docker_sandbox.go#L316